test/test_extract.py

```diff
@@ -118,14 +118,11 @@ def test_extract_file_rpm(self):
                 for e in self.extractor.file_extractors[self.extractor.extract_file_rpm]
             ]
         ):
-            if sys.platform.startswith("linux"):
+            print(os.path.join(extracted_path, "usr", "bin", "curl"))
```

Comment:
this should be a debug logging instead of a print, for consistency.

Reply:
It wasn't there and I just put it for debugging purpose and It isn't really necessary. so, I am removing it now.